react/modules/hideWhenUnavailable.ts

```diff
@@ -0,0 +1,24 @@
+interface Props {
+  showWhenUnavailable: boolean
+  AvailableQuantity: number
+}
+/**
+ * hideWhenUnavailable -> Logic to hide or show block based on props passed by store theme
+ *
+ * @export
+ * @param {Props} {
+ *   showWhenUnavailable,
+ *   AvailableQuantity,
+ * }
+ * @returns {boolean}
+ */
+export function hideWhenUnavailable({
```

Comment:
`showProduct`  or `isProductAvailable` is more intuitive, isn't it? I'm not sure, I can't just understand well, but the name of this predicate and what it's doing isn't aligned.

Reply:
How about `hideProductPrice`? This way we can keep the behavior of returning `true` when we need to hide the button, avoiding having to negate the return value inside the if statement.

Reply:
Sure!